README.adoc

```diff
@@ -53,6 +59,12 @@ Inject processing report messages into request metrics for analytics.
 ^.^|boolean
 ^.^|false
 
+.^|straightRespondMod
```

Comment:
Hum... I'm asking myself about this...

If `straightRespondMod` is set to `true`, what is the purpose of defining policy on the response scope ? I mean, what is the diff between `straightRespondMod` and not configuring the policy on the response scope?

Reply:
Oops I forgot to mention that in description, but when you set `straightRespondMod` to `true` validation failures messages are still being written to the metrics. So, it`s posiible to see the problems and fix them without affecting the external user.